testsuite/MDAnalysisTests/coordinates/test_gro.py

```diff
@@ -260,13 +260,16 @@ def test_check_coordinate_limits_max_noconversion(self):
 
 
 class TestGROWriterLarge(TestCase, tempdir.TempDir):
-    def setUp(self):
-        self.tmpdir = tempdir.TempDir()
-        self.large_universe = mda.Universe(GRO_large)
 
-    def tearDown(self):
-        del self.tmpdir
-        del self.large_universe
+    @classmethod
```

Comment:
I've switched to classmethods so that the setup and teardown operations are not repeated for each individual test -- only once at the start and once at the end for the tests in this class.